lib/Sema/TypeCheckProtocol.cpp

```diff
@@ -5124,7 +5124,7 @@ hasInvalidTypeInConformanceContext(const ValueDecl *requirement,
 }
 
 void ConformanceChecker::resolveValueWitnesses() {
-  for (auto member : Proto->getMembers()) {
+  for (auto member : Proto->getABIMembers()) {
     auto requirement = dyn_cast<ValueDecl>(member);
```

Comment:
It looks like we iterate over members in a few places and filter out those that are not ValueDecl or isProtocolRequirement(). Maybe we should add a ProtocolDecl::getProtocolRequirements() method (you don't have to do it in this PR though)

Reply:
Added!